buku.py

```diff
@@ -3211,6 +3222,7 @@ def main():
     power_grp = argparser.add_argument_group(
         title='POWER TOYS',
         description='''    --ai                 auto-import from Firefox and Chrome
+    --colors             set output colors
```

Comment:
`--colors COLORS     set output colors`

and please move it back to just before `--nc`